pages/api/lockerboxes/validateqr.ts

```diff
@@ -0,0 +1,42 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+import prisma from '../../../lib/prisma'
+import * as argon2 from 'argon2'
+import { Status } from '../../../types/types'
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === 'POST') {
+    try {
+      const { qrCode } = req.body
+      let unfilled_prescriptions = await prisma.prescription.findMany({
```

Comment:
use camel case for javascript
```suggestion
      const unfilledPrescriptions = await prisma.prescription.findMany({
```